pypac/api.py

```diff
@@ -256,7 +264,8 @@ def do_proxy_failover(self, proxy_url, for_url):
         self._proxy_resolver.ban_proxy(proxy_url)
         return self._proxy_resolver.get_proxy_for_requests(for_url)
 
-    def get_pac(self):
+    def get_pac(self, url=None, js=None, from_os_settings=True, from_dns=True,
+                timeout=2, allowed_content_types=None, **kwargs):

```

Comment:
How about doing `def get_pac(**kwargs)` to avoid duplicating the signature and defaults of the `get_pac()` function?

Reply:
Done. But I sink that using `**kwargs` is not good for code completion :(